README.md

```diff
@@ -9,3 +9,29 @@ names and labels by using a set of regular expressions.
 
 Please refer to this utility's `main()` function for a list of supported
 command line flags.
+
+- - - -
+
+# Building
+
+To build this code, you'll need a working `go` environment. Once you have a suitable environment, simply run the following commands
```

Comment:
Instead of explaining how to build it manually, I think it would make more sense to document `go install github.com/kumina/unbound_exporter`.